### ISSUE TITLE
Docs - FAQ Suggestion

### ISSUE THREAD

**Issue by yvrdarb** [CIDRAM/Docs#3]
A handy addition to the FAQs would be " How to Unblock Yourself" as I am currently locked out.

Thanks!

**Comment by Maikuolan** [comment]
Good suggestion. :-)

**Comment by Maikuolan** [comment]
@yvrdarb I forgot to ask when I replied earlier: Are you still currently locked out? (Got the suggestion, but missed the possible implied support angle there; Sorry about that).

**Comment by Maikuolan** [comment]
(And I'll start working on the new FAQ entry soon, when I've got a moment. Hoping to tag a new version/release either today or tomorrow, so working on that at the moment).

**Comment by yvrdarb** [comment]
No, thanks for asking.  I just saved my config file and nuked everything; it is a pretty quick rebuild after that.  Maybe also add the suggestion to whitelist your IP in the installation directions also .... ;)

**Comment by yvrdarb** [comment]
FYI, the upgrade wiped my config file today.  I just uninstalled it as a plugin and did a manual install and used a backup config.
